packages/core/src/Modules/Cashier/Components/Error/virtual.jsx

```diff
@@ -15,18 +16,34 @@ class Virtual extends React.Component {
     render = () => {
         return (
             <div className='cashier__wrapper'>
-                <h2 className='cashier-error__header cashier__virtual-header'>
+                {this.props.has_real_account && (
+                    <div
+                        className={classNames(
+                            'cashier__account-switch-icon',
+                            this.props.is_dark_mode_on
+                                ? 'cashier__account-switch-icon--dark'
+                                : 'cashier__account-switch-icon--light'
```

Comment:
usually we will use one svg and pass different classes to its fill colour to specify different colours, we can't do that here?

Reply:
@negarn these are used as CSS background images (can't pass classes to background images) and also there are 2 separate files in Zeplin. you mean I remove the background image and use the SVG directly inside markup and combine 2 SVGs manually to support light and dark theme in one file with different classes?

Reply:
@mahdiaryayi-fs you can look at some of our existing icons like ic-checkmark-circle.svg which is using `var(--fill-color1)` as the fill colour so it can change dynamically per light/dark theme and how it is imported to be used as part of the `<Icon />` component. we try to store only one image and make it responsive this way, there are a number of fill classes that you can use to make light/dark theme match closely to the zeplin design. if they are too different we can consider storing two individual files for them.

Reply:
Regarding our talk in Slack, I decided to keep both SVGs because of complicated colors (in comparison with icons)